### ISSUE TITLE
Mobile/Staging: Search bar on homepage takes up too much space

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#666]
![img_5409](https://cloud.githubusercontent.com/assets/5550969/16330570/db221952-39f2-11e6-9ceb-e2ba091a065f.PNG)

Something is going on with the search bar on mobile. It is taking up most of this page's space.

**Comment by theandrewbriggs** [comment]
This bug is fixed on staging.

**Comment by smarziano** [comment]
Confirmed that this is fixed on staging. Adding "Waiting on Merge" label.

@theandrewbriggs Larger question is if we want to keep the search bar on the main image on the homepage for mobile versions. We talked about how it might be better to move it so as not to obscure the image.

**Comment by smarziano** [comment]
This is fixed and live. New issue #677 created to work on improvements to mobile.
